builder.go

```diff
@@ -317,7 +318,7 @@ func (e *Expression) build(spacing string) string {
 		for _, o := range e.function.options {
 			s += " " + o.key + spacing + "=" + spacing
 
-			if v, ok := o.operand.(string); ok && len(v) > 0 && v[0] == '$' { // recognized as variable if string starts with $
+			if v, ok := o.operand.(string); ok && len(v) > 0 {
```

Comment:
It might fail on quoted literals after removal of - `&& v[0] == '$'`

Reply:
Also, the option value is not optional. E.g., an empty string should be `host = ||`.

Reply:
We don't need to write the dollar symbol before the variable anymore, so why will it fail?